google/cloud/security/common/data_access/forseti.py

```diff
@@ -254,6 +257,33 @@ def __repr__(self):
                 self.name,
                 self.display_name)
 
+    if schema_number >= 2.0:
```

Comment:
Is this ```if``` condition necessary here?  You seem to have implemented this on the caller below, which is a good thing in general.